content/blog/production-ready-object-detection-model-training-workflow-with-hpe-machine-learning-development-environment.md

```diff
@@ -0,0 +1,1099 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: model-store-pod
+spec:
+  volumes:
+    - name: model-store
+      persistentVolumeClaim:
+        claimName: task-pv-claim
+  containers:
+    - name: model-store
+      image: ubuntu
+      command: [ "sleep" ]
+      args: [ "infinity" ]
+      volumeMounts:
+        - mountPath: "/pv"
+          name: model-store
+      resources:
+        limits:
+          memory: "1Gi"
+          cpu: "1"
+```
+
+To create the Pod, run the command: `kubectl apply -f k8s_files/model-store-pod.yaml`
+
+## Preparing custom model for Kserve inference
+
+Here we will complete some preparation steps to deploy a trained custom FasterRCNN Object Detection model using KServe. A pre-requisite is to download a checkpoint from a determined experiement. You can read this [tutorial](https://docs.determined.ai/latest/training/model-management/checkpoints.html#downloading-checkpoints-using-the-cli) on how to download a checkpoint using the Determined CLI. For this tutorial, you can download an already prepared checkpoint using the following bash command:
+
+* `wget -O kserve_utils/torchserve_utils/trained_model.pth https://determined-ai-xview-coco-dataset.s3.us-west-2.amazonaws.com/trained_model.pth`
+
+### Stripping the Checkpoint of the Optimizer State Dict
+
+Checkpoints created from a Determined Experiment will save both the model parameters and the optimizer parameters. We will need to strip the checkpoint of all parameters except the model parameters for inference. Run the bash command to generate `train_model_stripped.pth`:
+
+Run the below command in a terminal:
+
+```cwl
+python kserve_utils/torchserve_utils/strip_checkpoint.py --ckpt-path kserve_utils/torchserve_utils/trained_model.pth \
+  --new-ckpt-name kserve_utils/torchserve_utils/trained_model_stripped.pth
+```
+
+### Run TorchServe Export to create .mar file
+
+Run the below command to export the Pytorch Checkpoint into a .mar file that is required for torchserve inference. Our Kserve InferenceService will automatically deploy a Pod with a docker image that support TorchServe inferencing.
+
+```cwl
+torch-model-archiver --model-name xview-fasterrcnn \
+  --version 1.0 \
+  --model-file kserve_utils/torchserve_utils/model-xview.py \
+  --serialized-file kserve_utils/torchserve_utils/trained_model_stripped.pth \
+  --handler kserve_utils/torchserve_utils/fasterrcnn_handler.py \
+  --extra-files kserve_utils/torchserve_utils/index_to_name.json
+```
+
+After command finishes, run command to move file to our prepared `model-store/` directory:
+
+* `cp xview-fasterrcnn.mar kserve_utils/torchserve_utils/model-store -v`
+
+### Copy `config/` and `model-store/` folders to the K8S PVC Pod
+
+This is the directory structure needed to prepare our custom Pytorch Model for KServe inferencing:
+
+```
+├── config
+│   └── config.properties
+├── model-store
+│   ├── properties.json
+│   └── xview-fasterrcnn.mar
+```
+
+#### What the config.properties file looks like
+
+```
```

Comment:
I would suggest here to specify the language syntax as: markdown.
This will allow the CMS editor to render the structure of the config.properties file instead of getting a everything in a single line.